docker-entrypoint.sh

```diff
@@ -40,9 +40,13 @@ if (is_numeric($socket)) {
 	$socket = null;
 }
 
-$maxTries = 10;
+$maxTries = 20;
 do {
-	$mysql = new mysqli($host, YOURLS_DB_USER, YOURLS_DB_PASS, '', $port, $socket);
+
+        $mysql = mysqli_init();
+        if( file_exists("/etc/ssl/certs/db-ca.crt") ) { $mysql->ssl_set(NULL, NULL, "/etc/ssl/certs/db-ca.crt", NULL, NULL); }
```

Comment:
`/etc/ssl/certs/db-ca.crt` might or might not exist. Is that path has to be specified?

Reply:
The docker container needs a static path, unless you want to implement an environment variable.

The "file_exist" function gives non SSL people functionality.

Otherwise it needs to be documented on the docker hub how to pass your ca.crt file using a docker volume mount.

Reply:
> unless you want to implement an environment variable

Really good suggestion, seems much more appropriate!

Reply:
The only problem with an environment variable, is that you would need the environment variable for the path as well as a volume mount for the ca.crt.

It would be cleaner, imo, to simply mount a ca.crt to a specific file/directory and check that file to exist (as done in my suggested pull)

But I think either method is fine as long as it's documented.